packages/react-components/src/alert/alert.tsx

```diff
@@ -98,11 +98,55 @@ const StyledAlert = styled(Box, {
           color: '$warning600'
         }
       }
+    },
+    variant: {
+      filled: {
+        backgroundColor: 'currentColor',
+        color: '$white900',
+        border: 'none',
+        [`${StyledAlertIcon} svg`]: {
+          color: '$white900'
+        }
+      },
+      default: {
+        unset: 'none'
+      }
     }
   },
+  compoundVariants: [
+    {
+      status: STATUSES.success.name,
+      variant: 'filled',
```

Comment:
@tmbalagan Our existing alert component already has a background color thereby making it a **filled**  variant already. It's just that the background color is of a lower opacity, which was taken as a design choice.  Doesn't it feel confusing to add another variant called filled ?

Reply:
Hi @tmbalagan, these new variants will certainly be helpful, especially in dark theme scenarios, as you mentioned.

However, as of now, we haven't integrated dark theme support across the entire component library. The alert component, along with other components, will be updated once we roll out full dark theme support. This will include introducing a new color palette specifically designed for dark mode, which will be an addition to the existing theme framework.

Your work on the alert component will definitely be useful when we get to that stage, and we appreciate your effort and foresight in preparing for dark theme compatibility. We’ll be sure to keep your contribution in mind when we move forward with the dark theme implementation.

Reply:
Thanks for the update! It’s great to know that my contributions will be useful for the dark theme. Looking forward to the next steps!